config/packages/test/monolog.yaml

```diff
@@ -1,7 +1,12 @@
 monolog:
     handlers:
         main:
+            type: fingers_crossed
```

Comment:
fingers_crossed? :grin:

Reply:
https://symfony.com/doc/current/logging.html#handlers-that-modify-log-entries :)